From 578e2676aa44dd439b98dcec383b77a7e5dbe369 Mon Sep 17 00:00:00 2001
From: Jiangli Yuan <jlyuan@motorola.com>
Date: Fri, 19 Feb 2016 20:42:07 -0800
Subject: [PATCH 687/959] Revert "IKSWM-21322 config: 8952: enable mmc deferred
 resume"

This reverts commit 724959bab27d8cf6dc82d5824af42d9aeed9f1f5.

Signed-off-by: Jiangli Yuan <jlyuan@motorola.com>

Conflicts:
	arch/arm/configs/ext_config/moto-msm8916.config

Change-Id: I4d223bf208972a212ed4425fa56bc7dabe0c8c01
Reviewed-on: http://gerrit.mot.com/823819
SME-Granted: SME Approvals Granted
SLTApproved: Slta Waiver <sltawvr@motorola.com>
Submit-Approved: Jira Key <jirakey@motorola.com>
Tested-by: Jira Key <jirakey@motorola.com>
Reviewed-by: Sheng-Zhe Zhao <a18689@motorola.com>
---
 arch/arm/configs/ext_config/moto-msm8916.config | 1 -
 1 file changed, 1 deletion(-)

diff --git a/arch/arm/configs/ext_config/moto-msm8916.config b/arch/arm/configs/ext_config/moto-msm8916.config
index 2246b54b820..689223f51fb 100644
--- a/arch/arm/configs/ext_config/moto-msm8916.config
+++ b/arch/arm/configs/ext_config/moto-msm8916.config
@@ -68,5 +68,4 @@ CONFIG_F2FS_FS=y
 CONFIG_F2FS_FS_SECURITY=y
 CONFIG_F2FS_FS_XATTR=y
 CONFIG_QPNP_USER_RESET=y
-CONFIG_MMC_BLOCK_DEFERRED_RESUME=y
 CONFIG_MMC_CQ_HCI=y
-- 
2.11.0

